test/permissions.test.js

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable no-unused-expressions */
+/* eslint-disable no-undef */
+import { Uuid } from '../packages/snjs/dist/@types/uuid.js';
+import * as Factory from './lib/factory.js';
+chai.use(chaiAsPromised);
+const expect = chai.expect;
+
+describe('permissions', async function () {
```

Comment:
Can we have these tests on `packages/snjs/test`? 🙂 See https://github.com/standardnotes/snjs/commit/bc7f114131241b14022c77484e8ffe3d31528950

Reply:
Sure thing @johnny243, thank you for migrating them! I'll merge your branch with this one